plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -323,11 +323,13 @@ module.exports = {
       return newState;
     }
 
-    this.volumeMounts =
-        volumeMountsReducer(this.volumeMounts, {type, path, value});
+    this.volumeMounts = volumeMountsReducer(
+      this.volumeMounts,
+      {type, path, value}
+    );
 
     newState = state.map((container, index) => {
-      if (this.volumeMounts.length !== 0) {
+      if (this.volumeMounts.length === 0) {
```

Comment:
I'm not sure why did you change the condition? When `volumeMounts.length === 0` we don't have anything to filter nor map. Do we?